perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/gcsfuse_mount.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
please remove this file (and from inside Dockerfile) if you follow the comment in resnet_run.sh. Also, please use log rotation as we decided to keep log rotation. You can build gcsfuse from [log_rotation](https://github.com/GoogleCloudPlatform/gcsfuse/tree/log_rotation) branch.

Reply:
Done